api/src/models/notes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Schema representing notes
+ */
+const mongoose = require('mongoose');
+
+const actions = {
```

Comment:
Since we're using this as an enum we can freeze it as such `const actions = Object.freeze({ ... });`